### ISSUE TITLE
How can I change variable on the layer ( fine tuning )

### ISSUE THREAD

**Issue by abdou31** [tensorflow/tensorflow#33332]
------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10 x64
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**:1.13.1
- **Python version**:3.6.4
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:9.0/7.4
- **GPU model and memory**:Nvidia Geforce 840m
- **Exact command to reproduce**: N/A

### Describe the problem
I have a pre-trained model that detects the whole face with 68 key points, I would like to use this pre-trained model to train my dataset for detecting only the eye region ( 40 points ).
So I need to change the value of units on the logits layer from 136(68 * 2) to 80(40 * 2).
I see an answer on Stackoverflow that I should use the fine-tuning, how can I use this technique to change some variables on the frozen model (.pb) ?

![mdlrp](https://user-images.githubusercontent.com/19480228/66752207-2f22f700-ee91-11e9-802c-3a40c540078d.PNG)

Hope I get an answer.

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!

**Comment by abdou31** [comment]
I have posted this question on Stackoverflow and I have tagged #Tensorflow on the question but I didn't get any answer.

**Comment by jvishnuvardhan** [comment]
@abdou31 As you mentioned you have a dataset for training, you could load the pre-trained model,  change layer properties (from non-trainable to `trainable`) of last few layers, add another layer if required, update last layer for correct number of outputs and then train the model with your data. Thanks!

**Comment by abdou31** [comment]
How can I change the layer properties of a frozen graph inference model ( .pb )?

**Comment by jvishnuvardhan** [comment]
@abdou31 Is this still an issue for you? Do you have access to the pre-trained model? Can you share a standalone code to reproduce the issue?

If this was resolved for you, then please feel free to close the issue. Thanks!

**Comment by jdduke** [comment]
This is not currently possible, at least not easily. You'll need the source graph to properly fine tune the model.
